tests/NCronJob.Tests/TimeZoneTests.cs

```diff
@@ -20,14 +20,14 @@ public void ShouldAssignCorrectTimeZoneToJobOptions()
     }
 
     [Fact]
-    public void ShouldDefaultToUtcIfTimeZoneNotSpecified()
+    public void ShouldNotInferAnythingIfTimeZoneNotSpecified()
     {
         var builder = new JobOptionBuilder();
         builder.WithCronExpression(Cron.AtEveryMinute);
 
         var options = builder.GetJobOptions();
 
-        options.Single().TimeZoneInfo.ShouldBe(TimeZoneInfo.Utc);
+        options.Single().TimeZoneInfo.ShouldBeNull();
```

Comment:
No impact on the user.

The timezone handling has been pushed to the JobDefinition as well.